docker/docker-compose.centos-6.116.yaml

```diff
@@ -6,7 +6,7 @@ services:
     image: netty:centos-6-1.16
     build:
       args:
-        java_version : "adopt@1.16.0-1"
+        java_version : "16.0.2-zulu"
```

Comment:
Do we need Java 16 now?

Reply:
we have it still on the CI pipeline for now. I think we can remove it later tho

Reply:
Makes sense.